Lib/fontParts/world.py

```diff
@@ -3,7 +3,8 @@ def OpenFont(path, showInterface=True):
     Open font located at **path**. If **showInterface**
     is ``False``, the font should be opened without
     graphical interface. The default for **showInterface**
-    is ``True``.
+    is ``True``. Note: **path** can both be a string representing
+    a file path, or a font object.
```

Comment:
What kind of font object? From the test I see an UFOReader instance is accepted, but I don't generally consider that a font object, so I'm confused.

Reply:
Ah, fair point. I'm not sure how to word this. fontShell will take a `defcon` font object as a reference, but that is fairly fontShell specific. Perhaps just best to say 'file path' and leave it at that? Or, would it be better to go back to the `pathOrObject` syntax and update the documentation to say "a file path or the environment specific font object"?

Reply:
the object is a defcon font object. This is handy to wrap a defcon font object with fontParts. This is must not be a UFOReader object.

Reply:
if font object is also possible then it must be `pathOrObject` like in a `RFont` :) [see](https://github.com/robotools/fontParts/blob/master/Lib/fontParts/base/font.py#L24)

so actually only the docs string had te be changed... 

thanks